rasterio/rio/warp.py

```diff
@@ -135,8 +135,7 @@ def warp(
                         raise click.BadParameter('Required when using --bounds',
                             param='res', param_hint='res')
 
-                    xmin, ymin, xmax, ymax = transform_bounds(src.crs, dst_crs,
-                                                              *bounds)
+                    xmin, ymin, xmax, ymax = bounds
```

Comment:
Currently the help text states "Bounds are in the source coordinate reference system." Switching this to assume destination is a breaking change - any implications to this?